.github/workflows/QA.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       - name: Set up Python ${{ matrix.python }}
         uses: actions/setup-python@v5
         with:
+          # to update to 3.13 once it lands in GH image
```

Comment:
That's because ATM the GH image doesn't include Python3.13 so setting it would require a download/install that would slow down this quick step for no real value.

Reply:
Makes sense